package.json

```
@@ -4,19 +4,27 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "jest",
-    "start": "npm-run-all start-db start-api",
-    "debug": "npm-run-all start-db debug-api",
+    "prepare": "husky install",    
+    
+    "check-types": "tsc --noEmit",
+    "check-format:darwin:linux": "prettier --check 'src/**/*.{js,ts,json}'",
+    "check-format:win32": "prettier --check src/**/*.{js,ts,json}",
+    "eslint:darwin:linux": "eslint 'src/**/*.ts'",
+    "eslint:win32": "eslint src/**/*.ts",
+    "format:darwin:linux": "prettier --write 'src/**/*.{js,ts,json}'",
+    "format:win32": "prettier --write src/**/*.{js,ts,json}",
+    "lint:darwin:linux": "npm-run-all eslint:darwin:linux check-format:darwin:linux check-types",    
+    "lint:win32": "npm-run-all eslint:win32 check-format:win32 check-types",    
```

run-script-osを呼ぶのがないとだめっぽい(CIが落ちてる)
```suggestion
    "check-format": "run-script-os",
    "check-format:darwin:linux": "prettier --check 'src/**/*.{js,ts,json}'",
    "check-format:win32": "prettier --check src/**/*.{js,ts,json}",
    "eslint": "run-script-os",
    "eslint:darwin:linux": "eslint 'src/**/*.ts'",
    "eslint:win32": "eslint src/**/*.ts",
    "format": "run-script-os",
    "format:darwin:linux": "prettier --write 'src/**/*.{js,ts,json}'",
    "format:win32": "prettier --write src/**/*.{js,ts,json}",
    "lint": "run-script-os",
    "lint:darwin:linux": "npm-run-all eslint:darwin:linux check-format:darwin:linux check-types",    
    "lint:win32": "npm-run-all eslint:win32 check-format:win32 check-types",    
```